projects/components/src/titled-content/titled-content.component.ts

```diff
@@ -10,13 +10,18 @@ import { TitledHeaderControlDirective } from './header-controls/titled-header-co
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <div class="titled-content-container">
-      <div class="header" [htLayoutChangeTrigger]="this.shouldShowHeader">
-        <ht-label *ngIf="this.shouldShowTitleInHeader" [label]="this.title" class="title"></ht-label>
-        <ht-link [paramsOrUrl]="this.link" class="link" *ngIf="this.link">
+      <div class="header" [ngClass]="this.headerPosition" [htLayoutChangeTrigger]="this.shouldShowHeader">
+        <ht-label
+          *ngIf="this.shouldShowTitleInHeader"
+          [ngClass]="this.titleStyle"
+          [label]="this.title"
+          class="title"
+        ></ht-label>
+        <ht-link [paramsOrUrl]="this.link" class="link" [ngClass]="this.titleStyle" *ngIf="this.link">
           <ht-button
             [label]="this.linkLabel"
-            role="${ButtonRole.Primary}"
-            display="${ButtonStyle.Text}"
+            role="${ButtonRole.Quaternary}"
```

Comment:
Is this a new button role? How is it different than what we have?

Reply:
Yeah, it's been around for a bit, but not too long. It's a more subdued blue.

Reply:
We should combine the two styles. I don't think it makes sense to have another style. Separate from this PR though.

Reply:
So, can we keep it for now and see in the future if we can merge both button styles?